app/helpers.php

```diff
@@ -385,9 +385,9 @@ function wiki_url($page = 'Welcome', $locale = null)
     return route('wiki.show', $params);
 }
 
-function bbcode($text, $uid, $withGallery = false)
+function bbcode($text, $uid, $withGallery = false, $withImageDimensions = true)
 {
-    return (new App\Libraries\BBCodeFromDB($text, $uid, $withGallery))->toHTML();
+    return (new App\Libraries\BBCodeFromDB($text, $uid, $withGallery))->toHTML(false, $withImageDimensions);
```

Comment:
maybe part of constructor? (and using array combined with `withGallery` and `ignoreLineHeight`)